### PR TITLE
#472 suspense loading spinner

### LINKED ISSUE #472: The main suspense fallback should be a better spinner
The main loading component when the app is loading is just `"Loading..."` which is annoying me.

I also found a funny spinner so want to add it 😁

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #472 

It would be best if you load this and take a look before you look at the code..

It's the spinner when you first load the page!

### CODE REVIEW

**Comment by mark-prins** [comment]
ha! you beat me to it! I was going to do the same thing - the `loading` suspense fallback bothered me too 😁

File: packages/common/src/ui/components/loading/Biker/Biker.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      transform: 'rotate(0deg)',
+    },
+    '100%': {
+      transform: 'rotate(360deg)',
+    },
+  },
+} as const;
+
+const text = [
+  'One sec, I got hit by a bus...',
```
Comment (mark-prins) [inline packages/common/src/ui/components/loading/Biker/Biker.tsx:52]: I think if you put this out there, and ask for suggestions on telegram, we'd get 1,000 odd posts

Comment (mark-prins) [reply packages/common/src/ui/components/loading/Biker/Biker.tsx:52]: As it is, I was hitting refresh a dozen times trying to read them all 🤣

**Review by mark-prins — APPROVED** [review]
🥇  legend! that is just too perfect!

I had thought of putting in an animated something and really liked the random message idea that videostream uses; once again you have done it better than I could have! 🤣

File: packages/common/src/ui/components/loading/Biker/Biker.tsx (hunk 1 above)
Comment (joshxg) [reply packages/common/src/ui/components/loading/Biker/Biker.tsx:52]: That message is paying homage to our falling comrade William!

Look, the implementation isn't ideal but .. whatever!

Comment (joshxg) [reply packages/common/src/ui/components/loading/Biker/Biker.tsx:52]: Oh. I did forget to mention that I tried using `NODE_ENV` and it doesn't work (Was going to put a boring spinner when not in dev!) so we'll have to setup some build thing for some dev variable, I think. But, I thought not to use translations as we'd use a 'real' (🙄 ) spinner in prod (Or, maybe just no text 🤣 )

Comment (mark-prins) [reply packages/common/src/ui/components/loading/Biker/Biker.tsx:52]: it's not like the translations work in prod environment anyway! 🤣 
I was going to look at `NODE_ENV` - had a feeling that wasn't working. I'll make an issue!

**Comment by joshxg** [comment]
Thanks! 😁

**Pull request merged**